panel/tests/ui/template/test_editabletemplate.py

```diff
@@ -176,7 +176,7 @@ def test_editable_template_drag_item(page):
 
     md2_handle.drag_to(md2_handle, target_position={'x': 0, 'y': -50}, force=True)
 
-    wait_until(lambda: list(tmpl.layout) == [id(md2), id(md1)], page)
+    wait_until((lambda: list(tmpl.layout), lambda: [id(md2), id(md1)]), page)
```

Comment:
I'm testing this out as I want more information than currently given with `wait_until`

Reply:
Yes, had a go at trying to get pytest to understand `wait_until` recently, without luck.

Reply:
Also looked into it found this http://pybites.blogspot.com/2011/07/behind-scenes-of-pytests-new-assertion.html